size_comparison/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cargo build --release
```

Comment:
you should specify the target here and `--workspace` is also required.

Reply:
What is `--workspace`? I don't see its documentation. Why should I specify the target here? It's pulled from `.config/cargo`.

Reply:
You're right. I actually broke the configuration when switching the target for me.